provisioner/terraform/parse.go

```diff
@@ -96,6 +205,31 @@ func convertVariableToParameter(variable *tfconfig.Variable) (*proto.ParameterSc
 	return schema, nil
 }
 
+// Converts a Terraform variable to a managed variable.
+func convertTerraformVariableToManagedVariable(variable *tfconfig.Variable) (*proto.TemplateVariable, error) {
+	var defaultData string
+	if variable.Default != nil {
+		var valid bool
+		defaultData, valid = variable.Default.(string)
+		if !valid {
+			defaultDataRaw, err := json.Marshal(variable.Default)
+			if err != nil {
+				return nil, xerrors.Errorf("parse variable %q default: %w", variable.Name, err)
+			}
+			defaultData = string(defaultDataRaw)
```

Comment:
I'm not sure what `variable.Default` looks like, so wondering if the idea with json marshaling here is to turn a string value like `hello` into `"hello"`? Or maybe the purpose is entirely different 😄